web-components/src/components/form/BuyCreditsForm/BuyCreditsForm.tsx

```diff
@@ -0,0 +1,207 @@
+import { ChangeEvent, MouseEvent, Suspense, useState } from 'react';
+import { SubmitHandler } from 'react-hook-form';
+import { TextButton } from 'web-components/src/components/buttons/TextButton';
+import CheckboxLabel from 'web-components/src/components/inputs/new/CheckboxLabel/CheckboxLabel';
+import { CreditsAmount } from 'web-components/src/components/inputs/new/CreditsAmount/CreditsAmount';
+import { Radio } from 'web-components/src/components/inputs/new/Radio/Radio';
+import { RadioGroup } from 'web-components/src/components/inputs/new/RadioGroup/RadioGroup';
+import { Loading } from 'web-components/src/components/loading';
+import Title from 'web-components/src/components/typography/new/Title';
+import Form from 'web-marketplace/src/components/molecules/Form/Form';
+import { useZodForm } from 'web-marketplace/src/components/molecules/Form/hook/useZodForm';
```

Comment:
we should avoid importing things from web-marketplace into web-components
I believe this BuyCreditsForm belongs more inside of web-marketplace than web-components

Reply:
After moving the components to web-marketplace the errors are still there, and there are other stories that return the same error. These errors are present in the production Storybook too: https://regen-storybook.netlify.app/?path=/story/registry-organisms-bridge-modal--bridge-modal

Reply:
It could be related to https://github.com/regen-network/regen-web/blob/c71aa3332aebf59b6cb2233f430e6751e2cbac87/web-storybook/.storybook/main.js#L96

We may need the polyfill in other environments as well..

Reply:
Although the polyfill seems to be out of date and a new one is now recommended: https://github.com/remorses/esbuild-plugins#readme

Reply:
This is fixed now. The issue was that Buffer wasn't available in the Browser environment not with the polyfill.